From: Dave Plater <davejplater@gmail.com>
Date: Thu  8 Apr 10:29:11 SAST 2021
Subject: Remove test for audacity's fork of wxWidgets
References: Audacity-devel list
Upstream: Supplied via email

Audacity 3.0.0 has a test requiring their fork of wxWidgets 3.1.3
but all the fixes are in standard wxWidgets 3.1.4 as in openSUSE:Tumbleweed.
This patch removes the test.
Index: b/cmake-proxies/wxWidgets/CMakeLists.txt
===================================================================
--- a/cmake-proxies/wxWidgets/CMakeLists.txt	2021-03-09 12:19:38.000000000 +0200
+++ b/cmake-proxies/wxWidgets/CMakeLists.txt	2021-04-08 10:24:32.927995434 +0200
@@ -237,16 +237,6 @@ file(
       "^#define +wxVERSION_STRING +"
 )
 
-string( REGEX MATCHALL "\".+(Audacity).+\"" ours "${output}")
-if( NOT ours )
-   message( FATAL_ERROR
-      "\n########################################################################\n"
-      "Audacity version 3.0.0 or higher requires use of a customized version of "
-      "wxWidgets.  For details:\n"
-      "  https://wiki.audacityteam.org/wiki/Building_for_Distros\n"
-      "########################################################################\n"
-   )
-endif()
 
 target_include_directories( ${TARGET} INTERFACE ${INCLUDES} )
 target_compile_definitions( ${TARGET} INTERFACE ${DEFINES} )
